roles/download/defaults/main.yml

```diff
@@ -157,6 +159,7 @@ kubelet_checksums:
     v1.15.0: a16443434eb0391991bff9a10288d83beb38be4d406954858d9fdc1063870284
   amd64:
     v1.17.0: c2af77f501c3164e80171903028d35c632366f53dec0c8419828d4e55d86146f
+    v1.16.7: f49755b06848914c2729353d3580199a70ec8d732609660e90214b4f48ff4398
```

Comment:
I see `c1bbb9c29d06d183a87127c10ea87da47e8eb5cf7ab28b047985da537f6dd7f5` on my laptop with `scripts/download_hash.sh v1.16.7`

Reply:
Mmm  scripts/download_hash.sh doesn’t download things if there is a cached version already there. Cleaned up the cache and I have the same checksum now!